services/drupal/web/modules/custom/epa_web_areas/epa_web_areas.module

```diff
@@ -140,6 +140,7 @@ function epa_web_areas_field_widget_form_alter($element, FormStateInterface $for
   if (!empty($plugin_type)
       && $plugin_type == 'media_library_widget'
       && $form_state->getFormObject() instanceof EntityForm
+      && $form_state->getFormObject()->getEntity() instanceOf ContentEntityInterface
```

Comment:
@carsoncho I don't think we want to do it this way because it short circuits the other side of our conditional.  I think your approach here would fail on forms where we're creating a brand-new node and we have to try to extract the group context from the form_state storage.  Move your check down to the if statement around line 149.

Reply:
@azinck Done. I've moved that instanceOf check down further in the logic per your recommendation.